src/App.js

```diff
@@ -67,6 +67,17 @@ class Root extends React.Component {
       // We are still receiving lot of ws messages that are destined to the admin-frontend and not this wallet
       // TODO separate those messages
       if (wsData.type === 'wallet:address_history') {
+        // If is a new transaction, we send a notification to the user, in case it's turned on
+        if (!wallet.txExists(wsData.history)) {
+          const message = 'New transaction! Click to open'
```

Comment:
Maybe "You've received a new transaction."

Reply:
Changed